### DISCUSSION TITLE
バグ: Organization ページ内のメンバープロフィールリンクがおかしい

### DISCUSSION THREAD

**Discussion by sya-ri** [increments/qiita-discussions#136] [要望・アイデア・不具合]
## 問題

[![](https://user-images.githubusercontent.com/34268371/153071955-f68a45df-47ff-42c9-9cc5-789e450ca5a4.png)](https://qiita.com/organizations/qiita-inc)

メンバーアイコン一覧の左上にある Qiita をクリックすると https://qiita.com/organizations/Qiita に飛び 404 になってしまいます。

## 本来の挙動

https://qiita.com/Qiita に飛ぶのが正常な挙動だと思います。

**Comment by kyntk** [comment]
@sya-ri ご報告いただきありがとうございます。
こちら不具合のようなので、修正いたします 🙇🏽

**Comment by kyntk** [comment]
@sya-ri 不具合の修正完了しました！

**Comment by sya-ri** [comment]
対応いただきありがとうございます☺️
